creativecloud/blocks/interactive-marquee/interactive-marquee.js

```diff
@@ -74,4 +74,17 @@ function interactiveInit(el) {
 
 export default async function init(el) {
   interactiveInit(el);
+  switch (true) {
+    case el.classList.contains('firefly'): {
+      const buttons = el.querySelectorAll('.con-button');
+      if (buttons.length === 0) return;
+      buttons.forEach((button) => { if (button.innerText.includes('Firefly')) button.setAttribute('daa-ll', 'getfirefly'); });
```

Comment:
Maybe we should move the button attribute handle part or the processing for firefly condition in a function like 'handleFirefly' and call only that function here. Would help keep the init function clean given that there are going to be multiple more such marquees developed. Thoughts?

Reply:
done